extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -30,6 +30,8 @@ import {installCidService,} from
 import {urlReplacementsForDoc} from '../../../../src/url-replacements';
 import * as sinon from 'sinon';
 
+import {AmpDocSingle} from '../../../../src/service/ampdoc-impl';
```

Comment:
Any way to switch to `describes.fakeWin` or `realWin` - they pre-created ampdoc environment for you.